webpack-blocks.js

```diff
@@ -0,0 +1,88 @@
+const { entryPoint, group, setOutput, sourceMaps } = require('@webpack-blocks/webpack2');
```

Comment:
hmm, not really, maybe just putting all webpack related stuff to some subfolder, to make root "clean".

Also I'm not a huge fan of using magic things like webpack-block, but that's your call I guess. 

otherwise LGTM